app/scripts/background.js

```diff
@@ -345,12 +341,22 @@ function setupController(initState, initLangCode) {
    */
   function connectRemote(remotePort) {
     const processName = remotePort.name;
-    const isMetaMaskInternalProcess = metamaskInternalProcessHash[processName];
 
     if (metamaskBlockedPorts.includes(remotePort.name)) {
       return;
     }
 
+    let isMetaMaskInternalProcess = false;
```

Comment:
I don't think we need to initialize this value to `false` since the `else` below will set that value.

Reply:
The safe, explicit default is nice though. I think we should keep it!

Reply:
Cool with me